js/bitmart.js

```diff
@@ -445,20 +445,21 @@ module.exports = class bitmart extends Exchange {
         //         "trace":"a67c9146-086d-4d3f-9897-5636a9bb26e1",
         //         "data":{
         //             "symbols":[
-        //                 {
-        //                     "symbol":"PRQ_BTC",
-        //                     "symbol_id":1232,
-        //                     "base_currency":"PRQ",
-        //                     "quote_currency":"BTC",
-        //                     "quote_increment":"1.0000000000",
-        //                     "base_min_size":"1.0000000000",
-        //                     "base_max_size":"10000000.0000000000",
-        //                     "price_min_precision":8,
-        //                     "price_max_precision":10,
-        //                     "expiration":"NA",
-        //                     "min_buy_amount":"0.0001000000",
-        //                     "min_sell_amount":"0.0001000000"
-        //                 },
+        //               {
+        //                  "symbol": "BTC_USDT",
+        //                  "symbol_id": 53,
+        //                  "base_currency": "BTC",
+        //                  "quote_currency": "USDT",
+        //                  "base_min_size": "0.000010000000000000000000000000",
+        //                  "base_max_size": "100000000.000000000000000000000000000000",
+        //                  "price_min_precision": -1,
+        //                  "price_max_precision": 2,
+        //                  "quote_increment": "0.00001", // The minimum order quantity is also the minimum order quantity increment [Comment: I think they confuse term 'order quantity' for 'order cost', as price increment is 0.01 for BTC/USDT]
```

Comment:
Please, elaborate a bit on this comment. On a sidenote, let's use as few punctuation characters in comments as possible.

Bad:

```JavaScript
const a = 1; // [~*comment**~: hi (because, why not)]
```

Good:

```JavaScript
const a = 1; // hi
```

Reply:
for exmaple, for BTC/USDT , 
"quote_increment": "0.00001", 
while tickSize (minimum price change) is `0.01`.

so, I've just added a note to further reader, that `quote_increment` is not usable value. (also, it was neither implemented anywhere before this PR and neither I've implemented that value in any calculation).

Reply:
i've shortened the comment too :)